nuxeo-core/nuxeo-core-io/src/main/java/org/nuxeo/ecm/core/io/marshallers/json/enrichers/AbstractJsonEnricher.java

```diff
@@ -56,8 +64,20 @@ public boolean accept(Class<?> clazz, Type genericType, MediaType mediatype) {
 
     @Override
     public void write(Enriched<EntityType> enrichable, JsonGenerator jg) {
-        try {
-            write(jg, enrichable.getEntity());
+        try (Writer writer = new StringWriter()) {
```

Comment:
The `writer` is not used at all and can be removed, I guess this is old code from a previous attempt